spec/index.html

```diff
@@ -5364,7 +5364,7 @@ <h4>Operator Extensibility</h4>
       </section>
       <section id="SparqlOps">
         <h3>Function Definitions</h3>
-        <p>This section defines the operators and functions introduced by the SPARQL Query language.
+        <p>This section defines the operators and functions introduced by the SPARQL query language.
```

Comment:
I do *not* understand the variance in capitalization. `SPARQL Query language` here (line 5367) is being changed to `SPARQL query language`, while earlier in the document (line 393), `SPARQL query language` is being changed to `SPARQL Query Language`. Note that neither line contains (part of) a title; they're both body prose.

There are other variances elsewhere in the document. Why do these vary? What is being communicated by the difference in casing, besides confusion?

Reply:
Picked up in https://github.com/w3c/rdf-star-wg/issues/144